parentheses/algorithm.go

```diff
@@ -0,0 +1,48 @@
+// Package implements function BalancedString, that verifies if the given string is a balanceds sequence of brackets
+package parentheses
+
+// BalancedString verifies if the given string is a balanceds sequence of brackets
+func BalancedString(s string) bool {
+	list := []string{}
```

Comment:
1. rename `list` to `stack`
2. init your stack with capacity

Reply:
Done